### PR TITLE
refactor: various landing page components

### PR DESCRIPTION
**Pull request by Coyenn**
## ✅ Checklist

- [x] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-08-15).
- [x] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] I performed a functional test on my final commit

---

## Changelog

Improves the typography on the landing page.

### CODE REVIEW

**Comment by Coyenn** [comment]
Bildschirm­foto 2022-09-18 um 21 58 51

I think the tweets on the right should be made into a slider using https://swiperjs.com/

**Comment by juliusmarminge** [comment]
> Bildschirm­foto 2022-09-18 um 21 58 51
> 
> I think the tweets on the left should be made into a slider using https://swiperjs.com/

that section looks really odd imo, why is the title not on top?

**Comment by Coyenn** [comment]
> that section looks really odd imo, why is the title not on top?

I know it looks weird currently. I didn't get to finish it yesterday. My idea ist that there will only be one Tweet visible at a time. That Tweet will change very few seconds using an autoplaying slider.

Why is the heading not on top?
Because we need a bit more variety. Nearly all the headings are currently on top and it just looks boring.

I'm always open to other solutions/suggestions if you have any!

**Comment by nexxeln** [comment]
> > that section looks really odd imo, why is the title not on top?
> 
> I know it looks weird currently. I didn't get to finish it yesterday. My idea ist that there will only be one Tweet visible at a time. That Tweet will change very few seconds using an autoplaying slider.
> 
> Why is the heading not on top? Because we need a bit more variety. Nearly all the headings are currently on top and it just looks boring.
> 
> I'm always open to other solutions/suggestions if you have any!

That sounds like a good idea

**Comment by Coyenn** [comment]
Current swiper implementation is kinda wacky. I'm looking into ways of how to solve this.

File: www/src/components/landingPage/tweets/tweets.astro [hunk 1]
```diff
@@ -1,61 +1,88 @@
...
-</section>
+</PageSection>
+
+<script type="module" is:inline client:visible>
+  import "../../../../node_modules/swiper/swiper-bundle.css";
+  import {
+    Autoplay,
+    Pagination,
+    Swiper,
+  } from "../../../../node_modules/swiper";
```
Comment (juliusmarminge) [inline www/src/components/landingPage/tweets/tweets.astro:70]: These imports are probably why it doesn't work in deployments

Comment (Coyenn) [reply www/src/components/landingPage/tweets/tweets.astro]: The imports weren't working properly. I refactored it into a react component and now it works as intended. Only thing left to fix is the pagination.

Comment (juliusmarminge) [reply www/src/components/landingPage/tweets/tweets.astro]: Can you not use a normal import for the vanilla js lib?

**Comment by thelamina** [comment]
I think the cards on the community section needs more horizontal padding to stay responsive

Community section screenshot

**Comment by juliusmarminge** [comment]
image

Possible to make this responsive?

**Comment by Coyenn** [comment]
> image
> 
> Possible to make this responsive?

Possibly. I'll look into it.

**Comment by Coyenn** [comment]
Bildschirm­foto 2022-09-20 um 14 05 20

I've made it fully readable at least

**Comment by Coyenn** [comment]
We might want to merge this soon. I don't want to keep feature branches open for too long.

**Review by juliusmarminge — APPROVED** [review]
Lovely, it's coming along very nicely. Maybe these buttons are a bit small in comparison to the rest of the content? 

CleanShot 2022-09-20 at 15 40 09@2x

Another thing I noticed is we might want to consider making the title a bit wider so it fits on 3 rows? The `project` on a single line looks a bit off? Would this fit nicely?

The best way to setup an

optionated, fullstack,

typesafe Nextjs project

**Review by nexxeln — APPROVED** [review]
Looks amazing man great job.

**Comment by Coyenn** [comment]
> Another thing I noticed is we might want to consider making the title a bit wider so it fits on 3 rows? The `project` on a single line looks a bit off? Would this fit nicely?
> 
> The best way to setup an
> 
> optionated, fullstack,
> 
> typesafe Nextjs project

Sure.

About the buttons:
I was planning on creating a button component soon so this kind of fits in with my plan. How about we make them the size of the buttons on https://vercel.com/solutions/nextjs

**Comment by c-ehrlich** [comment]
Nitpicking, but imo

> The best way to setup an
> optionated, full-stack, typesafe
> Nextjs project

works better even though the last line is shorter than the others

**Pull request merged**